articles/techniques/type-model.md

```diff
@@ -123,24 +125,25 @@ newtype Complex = (Double, Double);
 ```
 
 This statement creates a new type which is effectively a label for a particular tuple type.
-Values of the new type are created by using the name of the type as a function:
+Values of the new type are created by calling the corresponding type constructor:
 
 ```
 let realUnit = Complex(1.0, 0.0);
 let imaginaryUnit = Complex(0.0, 1.0);
 ```
 
-As with ordinary tuple types, the constituting elements of a user-defined type can be accessed using deconstruction.
+In order to access the items in a user defined type, the tuple first needs to be "cast" to the corresponding tuple type. 
```

Comment:
I would use a different term than "cast". I would avoid the word "cast" at all costs...
I like "unwrap", but I could handle other terms. I just really don't want to use the word "cast" -- PTSD from C++...

Reply:
edited